pkg/apis/v1alpha1/workload.go

```diff
@@ -40,7 +40,10 @@ const (
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-
```

Comment:
nit: preserve the blank line between the markers and the struct

Reply:
Done